praw/models/reddit/modmail.py

```diff
@@ -75,6 +75,10 @@ def __init__(self, reddit, id=None,  # pylint: disable=redefined-builtin
     def _info_path(self):
         return API_PATH['modmail_conversation'].format(id=self.id)
 
+    def archive(self):
+        """Archive the conversation."""
```

Comment:
Could you provide an example in all these new method docstrings? Ideally _all_ methods should have at least one example showing usage (anyone reading should feel free to open PRs for methods which are missing said examples).

I'm going to merge without them, but it would be awesome if you could go through and add them where missing (I might have neglected to check for that before).